### PR TITLE
fixed the testplaybook to handle the exception better

### PR DESCRIPTION
**Pull request by omerKarkKatz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml [hunk 1]
```diff
@@ -130,7 +131,7 @@ tasks:
       key:
         simple: emails
       value:
-        simple: '["email@domain.com", "email@sub.test.com", "invalidmail"]'
```
Comment (sapirshuker) [inline Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml:133]: You deleted the "invalidmail" email from the context values, as a result the task "IsEmailAddressInternal include_subdomains" with the email "invalidmail" won't be checked in your new test playbook. 
please add the same test with the error path to the "IsEmailAddressInternal include_subdomains" task.

File: Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml [hunk 2]
```diff
@@ -444,13 +445,124 @@ tasks:
...
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: cf79b114-1a67-4115-84c6-615493c3d669
+    type: regular
+    task:
+      id: cf79b114-1a67-4115-84c6-615493c3d669
+      version: -1
```
Comment (sapirshuker) [inline Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml:533]: Did you execute `format`?

**Review by sapirshuker — CHANGES_REQUESTED** [review]
Great! please see my comments.

File: Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml (hunk 2 above)
Comment (omerKarkKatz) [reply Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml:533]: Yes

File: Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml (hunk 1 above)
Comment (omerKarkKatz) [reply Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml:133]: It was never checked.

**Review by sapirshuker — APPROVED** [review]
Great!

File: Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml (hunk 1 above)
Comment (sapirshuker) [reply Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml:133]: You deleted the 'invalidmail' from the Set command, as a result the 'invalidmail' wont be checked in task "IsEmailAddressInternal include_subdomains"

Comment (sapirshuker) [inline Packs/CommonScripts/TestPlaybooks/IsEmailAddressInternal_Test.yml:133]: You deleted the 'invalidmail' from the Set command, as a result the 'invalidmail' wont be checked in task "IsEmailAddressInternal include_subdomains"

**Pull request merged**